packages/client/src/pages/Blog.jsx

```diff
@@ -1,32 +1,87 @@
 
-export default function BlogPosts() {
-  const { loading, error, data } = useQuery(GET_BLOG_POSTS);
+export default function Blog() {
+  const { loading, error, data, refetch } = useQuery(GET_BLOG_POSTS_BY_CATEGORY, {
+    variables: { category: '', page: 1, pageSize: 9 }
+  });
+  const [tabValue, setTabValue] = useState(0);
+  const [searchValue, setSearchValue] = useState('');
+  const blogCategories = {
+    all: 'All',
+    depression: 'Depression',
+    anxiety: 'Anxiety',
+    addiction: 'Addiction',
+    bipolar: 'Bipolar'
+  };
 
   if (loading) return <p>Loading...</p>;
   if (error) return <p>Error : {error.message}</p>;
 
-  return data.blogPosts.data.map(({ id, attributes }) => (
-    <div
-      key={id}
-      style={{
-        backgroundColor: theme.palette.purple.light,
-        color: theme.palette.purple.contrastText,
-        padding: '0.5rem 1rem',
-        margin: '1rem auto',
-        borderRadius: '1em'
-      }}
-    >
-      <h3>{attributes.Title}</h3>
-      <p>Published at: {attributes.DatetimePublished}</p>
-      <hr />
-      <ReactMarkdown transformImageUri={processMarkdownImageUri}>{attributes.Body}</ReactMarkdown>
-      <br />
-    </div>
-  ));
+  const handleChangeTab = (_event, newValue) => {
+    setTabValue(newValue);
+    const newCategory = Object.values(blogCategories)[newValue];
+    refetch({ category: newCategory === 'All' ? '' : newCategory });
+  };
+
+  const handleChangeSearch = (event) => {
+    setSearchValue(event.target.value);
+  };
+
+  const handlePageChange = (_event, page) => {
+    refetch({ page });
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex' }}>
+        <Tabs value={tabValue} onChange={handleChangeTab}>
+          {Object.keys(blogCategories).map((category) => (
+            <Tab key={category} label={blogCategories[category]} />
+          ))}
+        </Tabs>
+        <Paper
+          component="form"
+          onChange={handleChangeSearch}
+          sx={{ p: '2px 4px', alignItems: 'center', display: 'flex', marginLeft: 'auto' }}
+        >
+          <IconButton sx={{ p: '10px' }}>
+            <SearchIcon />
+          </IconButton>
+          <InputBase placeholder="Search Blogs" value={searchValue} sx={{ ml: 1, flex: 1, width: 260 }} />
+        </Paper>
+      </Box>
+      <Box sx={{ my: 5, mx: 2 }}>
+        <span style={{ fontWeight: 'bold' }}>Latest</span>
+        <h1 style={{ marginTop: 8 }}>Discover Our Blog Posts</h1>
+        <p>Stay informed with our latest blog updates.</p>
+      </Box>
+      <Grid container rowSpacing={5} role="tabpanel">
+        {data.blogPosts.data.map((post) => (
+          <Grid item xs={12} sm={6} md={4} key={post.id}>
+            <BlogCard
+              id={post.id}
+              tag={post.attributes.Category}
+              title={post.attributes.Title}
+              description={'description'}
+              imgSource={post.attributes.CoverImage.data?.attributes.url}
+              author={'author'}
```

Comment:
May I have your permission to:
1. Go on strapi and add these fields (description and author) to the blog post model?
2. Make a 'Blog Post Cover Images' folder on the Strapi media library? haha

Reply:
1. Not sure about the `description`. I asked Karina and she said "just do title only". For `author`, is there a way to query strapi uploader's name rather than let them type in their name?
2. Sure, go ahead

Reply:
1a. Oh OK whoops, in that case could we remove the  'description' parameter from BlogCard and remove the placeholder text on the card?
1b. No idea, but the uploader is actually quite unlikely to be the author anyway (the Restore team will have one/a few designated "website people" that will be different from people writing content) so we probably wouldn't want to do that anyway
2. OK thanks!!

Reply:
1b. If that's the case, then sure we can add `author` to the model. I'll later add that to the query.

Reply:
Thank uu! I added 'Author' to the blog post attributes